VMDP doc set (#111284)
openshift/openshift-docs: modules/oadp-vmdp-supported-platforms.adoc

// Module included in the following assemblies:
//
// backup_and_restore/application_backup_and_restore/oadp-vmdp/oadp-vmdp.adoc

:_mod-docs-content-type: REFERENCE
[id="oadp-vmdp-supported-platforms_{context}"]
= Supported platforms for {oadp-short} VMDP

[role="_abstract"]

> Review the supported guest operating systems and architectures for the {oadp-short} VM data protection (VMDP) CLI. This helps you to verify that your VM environment is compatible.

Reviewer: ```suggestion
Review the supported guest operating systems and architectures for the {oadp-full} virtual machine data protection (VMDP) command-line interface. This helps you to verify that your VM environment is compatible.
```

239 characters

VMDP is built for link:https://access.redhat.com/articles/4234591[{VirtProductName} certified guest operating systems] on the following platforms:

.Supported guest operating systems
[cols="1,1",options="header"]
|===
|Guest operating system |Architectures

|Red Hat Enterprise Linux

> |x86_64, aarch64

Reviewer: ```suggestion
|x86_64, AArch64
```

I believe we should start with capitals AA in this case, proof https://redhat-documentation.github.io/supplementary-style-guide/#aarch64

|Microsoft Windows

> |x86_64, aarch64

Reviewer: ```suggestion
|x86_64, AArch64
```

I believe we should start with capitals AA in this case, proof https://redhat-documentation.github.io/supplementary-style-guide/#aarch64
